v2/cli/init_templates.go

```diff
@@ -0,0 +1,130 @@
+    defaultConfigFiles:
+      brigade.{{ .Language }}: |
+        {{ nindent 8 .Script }}
+    {{- else }}
+    git:
+      cloneURL: {{ .GitCloneURL }}
+    {{- end }}
+`)
+
+// nolint: lll
+var scriptTemplate = []byte(`{{- if eq .Language "js" }}const { events } = require("@brigadecore/brigadier");
+
+events.on("brigade.sh/cli", "exec", async event => {
+	let job = new Job("hello", "debian:latest", event);
+	job.primaryContainer.command = ["echo"];
+	job.primaryContainer.arguments = ["Hello, World!"];
+	await job.run();
+});
+
+events.process(); 
+{{- else }}import { events, Job } from "@brigadecore/brigadier"
+
+events.on("brigade.sh/cli", "exec", async event => {
+	let job = new Job("hello", "debian:latest", event)
+	job.primaryContainer.command = ["echo"]
+	job.primaryContainer.arguments = ["Hello, World!"]
+	await job.run()
+}
+
+events.process()
+{{- end }}
+`)
+
+// nolint: lll
+var notesTemplate = []byte(`This file was created by brig init and outlines next steps.
+
+1. Edit project.yaml as you see fit and submit it to Brigade using:
+     brig project create -f .brigade/project.yaml.
+
+2. If your project needs to make use of any secret/sensitive values, set them
+   one at a time using:
+     brig project secrets set --id {{ .ProjectID }} --set <key>=<value>
+
+   Or set secrets in bulk by modifying secrets.yaml, then submit them to
+   Brigade using:
+     brig project secrets set --id {{ .ProjectID }} --file .brigade/secrets.yaml
+
+   Secrets become available within your brigade.{{ .Language }} script using:
+     event.project.secrets.<key>
+
+   Take great care in how you use secrets so they are not leaked into worker or
+   job logs.
+
+   Note that Brigade keeps all jobs' environment variables secret, so it is safe
+   to inject your secrets into a job's environment. This action will not
+   implicitly expose your secrets. Take care that the jobs themselves don't leak
+   secrets into their logs.
+
+   DO NOT commit secrets to source control. brig init has already added
+   .brigade/secrets.yaml to your .gitignore file.
+
+{{- if not (eq .GitCloneURL "") }}
+
+3. Since your project expects to handled events using your brigade.{{ .Language}}
+    script found at git repository {{ .GitCloneURL }},
+    be sure to commit and push your brigade.{{ .Language }} script to that
+    location.
+{{- end }}
+`)
+
+var secretsTemplate = []byte(`## This file was created by brig init.
+##
+## Specify values for project secrets using key/value pairs below.
+##
+## You can load all these secrets into your project using
+## brig project secrets set --id {{ .ProjectID }} --file .brigade/secrets.yaml
+##
+## DO NOT commit these secrets to source control. brig init has already added
+## .brigade/secrets.yaml to your .gitignore file.
+##
+## Example keys/values:
+##
+# foo: bar
+# bat: baz
+`)
+
+// Creates content from a given template and struct
+func execTemplate(
+	templateBytes []byte,
+	fields interface{},
+) (bytes.Buffer, error) {
```

Comment:
Nit: There might be circumstances where returning (and subsequently passing around) a buffer makes sense for reasons of memory management, but I can't find a good reason to do it here. Returning `[]byte` instead of a `bytes.Buffer` might be more straightforward.